src/explorer/model/MavenProject.ts

```diff
@@ -146,6 +147,9 @@ export class MavenProject implements ITreeItem {
         const ret: ITreeItem[] = [];
         ret.push(new LifecycleMenu(this));
         ret.push(new PluginsMenu(this));
+        if (this.packaging !== "pom") { // parent pom does not have dependencies
```

Comment:
> parent pom does not have dependencies

I doubt that. I assume we can define dependencies in "pom" project (shared by subprojects/modules), but here I think the purpose is to hide the "Dependencies“ item as it's not much helpful for the "pom" project itself.